cmd/kube-scheduler/app/server.go

```diff
@@ -339,3 +285,70 @@ func WithPlugin(name string, factory framework.PluginFactory) Option {
 		return registry.Register(name, factory)
 	}
 }
+
+// SetupCommand creates a completed config and a scheduler based on the command args and options
+func SetupCommand(ctx context.Context, args []string, opts *options.Options, outOfTreeRegistryOptions ...Option) (*schedulerserverconfig.CompletedConfig, *scheduler.Scheduler, error) {
```

Comment:
nit: `Setup`, as this is not related to `cobra.Command` anymore.

Reply:
done